ogre2/src/Ogre2GpuRays.cc

```diff
@@ -661,7 +661,22 @@ void Ogre2GpuRays::Destroy()
     ogreCompMgr->removeWorkspace(this->dataPtr->ogreCompositorWorkspace2nd);
     this->dataPtr->ogreCompositorWorkspace2nd = nullptr;
   }
+
+  if (this->scene)
+  {
+    Ogre::SceneManager *ogreSceneManager = this->scene->OgreSceneManager();
+    if (ogreSceneManager == nullptr)
+    {
+      ignerr << "Scene manager not available. "
+             << "Unable to remove cameras and listeners" << std::endl;
```

Comment:
We don't print errors for other missing things in the `Destroy` function. Just checking that there's a reason to treat the scene manager differently.

Reply:
not really. I removed the error msg print line. [734dfed](https://github.com/gazebosim/gz-rendering/pull/698/commits/734dfed35d00b4dbac853c0d26fbe9cdd3e4d0d0)